src/gettext.cpp

```diff
@@ -44,6 +44,7 @@
 namespace bl = boost::locale;
 namespace
 {
+	//This mutex avoids the crash reported in bug #2013:
```

Comment:
Ideally this should describe the crash directly here as well. Otherwise if we move off github at some point in the future, there's no guarantee that this will be useful (similar to what happened with GNA - comments referenced GNA issues that were no longer accessible).

Reply:
is abcded4 better?